src/react-sortable-tree.js

```diff
@@ -471,6 +474,7 @@ ReactSortableTree.propTypes = {
     // This is an advanced option for complete customization of the appearance.
     // It is best to copy the component in `node-renderer-default.js` to use as a base, and customize as needed.
     nodeContentRenderer: PropTypes.any,
+    contentRenderer: PropTypes.func.isRequired,
```

Comment:
Progressive overrides would help me with some design ideas I have for my JSON editor.

In particular, I'd like to 3d-flip the tree node in-place to reveal an edit box. Right now I'm about to implement it into a modal - I'm just not liking my progress so far... 💩